lib/backend/k8s/resources/ippool.go

```diff
@@ -69,41 +69,56 @@ func (_ IPPoolConverter) NameToKey(name string) (model.Key, error) {
 }
 
 func (_ IPPoolConverter) ToKVPair(r CustomK8sResource) (*model.KVPair, error) {
-	t := r.(*thirdparty.IpPool)
-	v := model.IPPool{}
+	t := r.(*custom.IpPool)
 
-	_, err := ResourceNameToIPNet(t.Metadata.Name)
+	// Convert k8s resource name format (192-168-0-1-24) to CIDR.
+	cidr, err := ResourceNameToIPNet(t.Metadata.Name)
 	if err != nil {
 		return nil, err
 	}
 
-	err = json.Unmarshal([]byte(t.Spec.Value), &v)
-	if err != nil {
-		return nil, err
+	ipipInterface := ""
+	if t.Spec.IPIP != nil {
+		if t.Spec.IPIP.Enabled {
+			ipipInterface = "tunl0"
+		} else {
+			ipipInterface = ""
+		}
 	}
+
 	return &model.KVPair{
-		Key:      model.IPPoolKey{CIDR: v.CIDR},
-		Value:    &v,
+		Key: model.IPPoolKey{CIDR: *cidr},
+		Value: &model.IPPool{
```

Comment:
we don't have any common converter methods for IPPool, I can take the one in `client/` and move it to `converter/` and use it here